docs/redirects.yaml

```diff
@@ -0,0 +1,10 @@
+# Readthedocs redirects
+# See https://docs.readthedocs.io/en/stable/user-defined-redirects.html
+#
+# In order to prevent exposing the API token, PR jobs do not update the
+# redirects - changes to this file are only applied when they are merged
```

Comment:
It looks like there's a security concern here, which it is up to reviewers to enforce.

Please document what reviewers should accept and what they must reject. We select reviewers for their knowledge of cryptography and C, not of readthedocs and web security.

Can we have an automated check that guides reviewers (so that common cases are accepted automatically, obvious bad things are rejected automatically, and reviewers are alerted if there's something unusual)? Or do we expect to change this very rarely so any change to this file needs to be reviewed by a domain expert?

Reply:
There is no security concern for reviewers to enforce.

It's the other way round - the API token is not exposed when building PR jobs, therefore the redirects cannot be updated in PR jobs.

This comment is a note to let reviewers know that they shouldn't expect to see the redirects present in the PR job when reviewing updates to it. See [my comment](https://github.com/Mbed-TLS/mbedtls-docs/pull/91#issuecomment-1511007442) on the corresponding PR for mbedtls-docs for specifics.

Reply:
Oh, I see. Then I'm changing my change request to request a clarification of the comment. I understood it as: the code here is this way for a security reason. But actually, this comment is saying that the code here is not taken into account at a point where one might think it's taken into account. And even so I'm not sure that comment is relevant here: it would seem more relevant in the yaml file. In terms of wording, “functional limitation (no update in PR job) because functional limitation (no API token available in PR job) because security (not leaking the API token)” rather than “security concern, therefore functional behavior” would make it clearer.